nullaway/src/main/java/com/uber/nullaway/Config.java

```diff
@@ -292,4 +292,12 @@ public interface Config {
    *     similarly for {@code @RecentlyNonNull}
    */
   boolean acknowledgeAndroidRecent();
+
+  /**
+   * Checks if {@link LibraryModels} can override annotations on annotated source codes.
```

Comment:
"source code". It's an uncountable noun (like "water"), unless you are talking about "return codes" or "opcodes" or some other discrete set of codes.

Reply:
Fixed it fa145f21ebfb8aedd345af5d0e6a014a5af5c468